entry_tasks/89_sequence_type_checker/89_sequence_type_checker.py

```diff
@@ -0,0 +1,41 @@
+if __name__ == '__main__':
+    #x = '1, sss, none, false, True, None, False,        qqq      ,  2.0, (ssss, 1)'
+    #out_put = [(1, <class 'int'>), ('sss', <class 'str'>), (None, <class 'NoneType'>), (False, <class 'bool'>), (True,  <class 'bool'>), (None, <class 'NoneType'>), (False, <class 'bool'>), ('qqq', <class 'str'>), (2.0, <class 'float'>), (('ssss', 1), <class 'tuple'>)]
+    x = input('Input any number or string, or True, or None or tuple. Separated by comma:')
+    def get_type(item):
+        try:
+            return int(item), int
+        except:
+            try:
+                return float(item), float
+            except:
+                if item == 'false' or item == 'true':
+                    return item, bool
+                elif item == 'True' or item == 'False':
+                    return item, bool
+                if item == 'none' or item == 'None':
+                    return item, None
```

Comment:
I think if we do 
```py
item = '  TrUe   '
item = item.strip().lower()  # 'true'
```
we can avoid extra checks in your ifs
and also, when you do if elif block, better to continue it

if
elif
elif
elif
else